src/components/Widget/Widget.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+	FaCloudRain,
+	FaMoneyBill,
+	FaHourglassHalf,
+	FaSeedling,
+	FaStar,
+} from 'react-icons/fa6';
+import { Badge } from '@/components/ui/badge';
+import { IconType } from 'react-icons/lib';
+import { string } from 'zod';
```

Comment:
It seems like this is not used? Can you please remove if not needed?

Reply:
Thanks for flagging, hadn't spotted that!